packages/edit-site/src/components/save-hub/index.js

```diff
@@ -107,6 +108,7 @@ export default function SaveHub() {
 	const saveCurrentEntity = async () => {
 		if ( ! dirtyCurrentEntity ) return;
 
```

Comment:
In other places where we are using the unique id we are not dismissing, but in those cases the notice message is usually unique so obvious it has changed. In this instance, unless we dismiss the notice first it can appear like the subsequent saves may not have been completed as the snackbar appears not to change.